pkg/plugin/group/rollingupdate.go

```diff
@@ -120,17 +123,27 @@ func (r *rollingupdate) waitUntilQuiesced(pollInterval time.Duration, expectedNe
 				// the group, and/or documenting the expectations for plugin implementations.
 				switch r.scaled.Health(inst) {
 				case flavor.Healthy:
+					log.Info("waitUntilQuiesced", "msg", fmt.Sprintf("Node %s is Healthy", inst.ID))
```

Comment:
The convention for logging is something like
```
log.Info("waitUntilQuiesced - node healthy", "nodeID", inst.ID)
```

Also, I don't know if we need to have a logging statement for each case...  maybe just one statement before the switch?

```
health:= r.scaled.Health(inst)
log.Info("waitUntilQuiesced", "nodeID", inst.ID, "health", health)
switch health {
...
```

Reply:
I can update the format, my motivation for being more verbose in the logging was because `health` is just an `int`.  IMO the log messages are more obvious when they have messages like `Unknown`/`Healthy`/`Unhealthy` vs `0`/`1`/`2`. Thoughts?

Reply:
Ah ok makes sense.   Then we probably don't need the `fmt.Sprintf()` then... just something like
```
log.Info("waitUntilQuiesed", "health", "heathy", "nodeID", inst.ID)
```
in each one of the `case`s.